torch/nn/modules/batchnorm.py

```diff
@@ -45,7 +45,7 @@ def reset_parameters(self):
             self.bias.data.zero_()
 
     def _check_input_dim(self, input):
-        return NotImplemented
+        raise NotImplementedError
```

Comment:
It should also be `raise NotImplementedError()`. Otherwise you're raising a class as an error, not an instance